remo/sdk.py

```diff
@@ -336,10 +336,11 @@ def get_annotation_info(self, dataset_id: int, annotation_set_id: int, image_id:
         resp = self.api.get_annotation_info(dataset_id, annotation_set_id, image_id)
         return resp.get('annotation_info', [])
 
-    def get_annotation(self, dataset_id: int, annotation_set_id: int, image_id: int) -> Annotation:
+    def get_image_annotations(self, dataset_id: int, annotation_set_id: int, image_id: int) -> Annotation:
```

Comment:
@drewlr do we really need to rename this method?

Reply:
@vmanilo cuurent name is not very clear to me - I had no idea what it referred to. So unless we make it an hidden function, I'd say yes better to rename it

Reply:
@drewlr what about `list_image_annotations`?